### ISSUE TITLE
suggestion - provide the option to store CADU files when use live processing

### ISSUE THREAD

**Issue by haiyangdaozhang** [altillimity/SatDump#144]
Hello,

I found that the program saves the CADU data only when the baseband data is stored and then decoded offline. Is there an option to save CADU files during live decoding?

Thanks for this software!

**Comment by ZbychuButItWasTaken** [comment]
Which pipeline are you using? CADU should be saved

**Comment by haiyangdaozhang** [comment]
fengyun svissr and gk2a lrit

**Comment by ZbychuButItWasTaken** [comment]
Ah yes, we do not save the cadu on these as they are meant to be run long term (which would make the cadu massive). Howevee we can think of an option to save it @Aang23

**Comment by Aang23** [comment]
Indeed, currently pipelines that process things down to images directly do not save the CADU. Would you want to "simply" disable said final image processing or rather dump the cadu alongside decoding images?

**Comment by haiyangdaozhang** [comment]
My suggestion is to add a button that allows dump the cadu alongside decoding images.
